lms/djangoapps/grades/new/course_grade.py

```diff
@@ -317,7 +317,7 @@ def create(self, course, read_only=True):
         if not self._user_has_access_to_course(course_structure):
             raise PermissionDenied("User does not have access to this course")
         return (
-            self._get_saved_grade(course, course_structure) or
+            self.get_saved_grade(course, course_structure) or
```

Comment:
The only reason I'm hesitant to just call this method `get` is that, unlike `create` or `update`, it varies in return value based on whether a persisted value exists or not. Maybe `get_persisted`?